### PR TITLE
add examples, exclude .idea dir

### PR DESCRIPTION
**Pull request by mrjones-plip**
I wasn't sure about some of the sample values like passwords and UUIDs - don't want give people the wrong idea and use a bad, sample password in production - maybe exclude? Open to ideas!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,37 +86,37 @@ Request body:
 ```Accepts: application/json```
 ```json
 {
-  "docker_compose": {
-    "<file_name1>": "<file contents>",
-    "<file_name2>": "<file contents>"
-  }
+ "docker_compose": {
```
Comment (dianabarsan) [inline README.md:89]: Are the indentation removals on purpose?

File: README.md [hunk 2]
```diff
@@ -25,33 +25,33 @@ Save this file in a new folder on your computer - a different folder than the on
...
+| Name | Required | Description | Example |
+|------|----------|-------------|---------|
+| `CHT_COMPOSE_PATH` | yes | Absolute path to the folder where the two CHT docker-compose files are saved. | `/home/cht` |
+| `COUCHDB_USER` | yes | CouchDB main admin account username. | `admin` |
+| `COUCHDB_PASSWORD` | yes | CouchDB main admin account password. | `thump-quake-agony-kite-civil-surf-mama` |
+| `COUCHDB_SECRET` | yes | CouchDB secret used by peers to communicate and to generate authentication cookies. Mandatory for CouchDB clustered mode to synchronize authentication cookie between nodes. | `kung-botch-pants-niece-lady-quill-elbow` |
+| `COUCHDB_UUID` | no | The UUID of the CouchDB Server used in [identifying the cluster when replicating](https://docs.couchdb.com/en/stable/setup/cluster.html#preparing-couchdb-nodes-to-be-joined-into-a-cluster) | `60c9e8234dfba3e2fdab04bf92001142` |
+| `COUCHDB_DATA` | yes | Absolute path to the folder that will serve as CouchDB data location. | `/home/cht/srv` |
+| `COUCHDB_SERVERS` | no | Comma separated list of all CouchDB services. Defaults to `couchdb`. | `couchdb.1,couchdb.2,couchdb.3` |
```
Comment (dianabarsan) [inline README.md:36]: Since this is aimed for self hosters, which won't have couchdb clusters, I think the example should be for a single node, not three.

**Review by dianabarsan — COMMENTED** [review]
Nice! So quick! Just two small comments.

File: README.md (hunk 1 above)
Comment (mrjones-plip) [reply README.md]: yikes! nice catch.  IDE autopilot?  Dunno.

reverted in cd36379

**Review by dianabarsan — APPROVED** [review]
Great! Thanks a lot!

**Pull request merged**